### DISCUSSION TITLE
WeeChat

### DISCUSSION THREAD

**Discussion by DakshG07** [catppuccin/catppuccin#1862] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
No, but I'm sure someone can.

**Additional context 📷 (optional)**
https://weechat.org/
Themes are found at: https://weechat.org/themes/

**Comment by Dinth** [comment]
Looks like weechat-android already has it: https://github.com/lokesh-krishna/weechat-android/blob/main/catppuccin-weechat.properties , but still missing a desktop version :(
